float32 acc
bool always
---
int8 Result